docs/ballot_encoding.md

```diff
@@ -0,0 +1,67 @@
+# Encoding of a ballot
+
+
+Here is described a common enconding for ballots, which is needed
+to make sure that all ballots can be encoded in a unique way (up to the
+ordering of questions).
+
+## Answers
+The answers to questions are encoded in the following way:
+
+```
+type:ID:<answers>
+```
+Where we have that:
+* the type can either be "select", "rank" or "text"  
+* ID is the ID of the question 
```

Comment:
but must have a common encoded size, right ?

Reply:
2 bytes encoded in base64 (4 chars)